test/api/BillingTest.ts

```diff
@@ -112,6 +114,8 @@ describe('Billing Service', function() {
   this.timeout(1000000);
   describe('With component Billing (tenant utbilling)', () => {
     before(async () => {
+      global.database = new MongoDBStorage(config.get('storage'));
```

Comment:
Ideally you should test that global.database is falsy, but nobody does that in the unit tests ;)

Reply:
I can approuve it but the merge will be done by Serge when his time permits, unless the rules have changed and now you also have the rights to do merge after one approval by me.